src/routes/protected.tsx

```diff
@@ -131,7 +132,17 @@ export const protectedRoutes: RouteNode[] = [
         path: Routes.PROJECT,
         element: <ProjectDashboard />,
         children: [
-          { path: '', element: <Navigate to='overview' replace /> },
+          {
+            path: '',
+            element: (
+              <ProjectRoute
+                permissions={['canViewEnrollmentDetails']}
```

Comment:
I had more questions about which is the correct permission to use here. I gathered a list of all the permissions that I thought might be relevant:

- Permissions on the project (`ProjectAccessFieldsFragment`)
  - `canEditEnrollments`
  - `canViewEnrollmentDetails`
  - (No such thing as simply `canViewEnrollments`)
- Other possibly relevant permissions on the HMIS admin dashboard
  - `canViewLimitedEnrollmentDetails` - this does not seem to be attached to a project, but if someone has this permission overall, should they be directed to a project enrollment page, or to the overview?
  - `canViewOpenEnrollmentSummary` which I think is associated with the Root permissions - not associated with a particular project